app-connector-client/server/services.js

```diff
@@ -88,6 +159,26 @@ function updateService(api, api_id, hostname) {
     })
 }
 
+function fillEventData(event) {
+    var specInJson
+    if (event.specification.endsWith(".json")) {
+        specInJson = JSON.parse(fs.readFileSync(event.specification))
+    } else {
+        specInJson = yaml.safeLoad(fs.readFileSync(event.specification, 'utf8'))
+    }
+
+    var serviceData = {
+        provider: event.provider ? event.provider : "Varkes",
+        name: event.name,
+        description: event.description ? event.description : event.name,
+        labels: event.labels ? event.labels : {},
+        type: "Event",

```

Comment:
What about using the spec type instead of that custom enum?

Reply:
but odata doesn't have that

Reply:
As discussed it will be changed in a different PR